src/components/PageHeader/index.js

```diff
@@ -45,20 +52,24 @@ const PageHeader = ({ className, children, title }) => {
     };
   });
 
+  const StyledHeading = styled(Heading)`
+    font-size: 20px;
+  `;
```

Comment:
you should also add line-height I think

Reply:
`line-height` Has already been provided by the `asc-ui` package. Don't need to set it again.